package.json

```diff
@@ -53,5 +53,8 @@
     "js-yaml": "^3.13.1",
     "lodash": "^4.17.20",
     "node-notifier": "^8.0.1"
+  },
+  "resolutions": {
+    "d3-color": "3.1.0"
```

Comment:
There is also a reference to the d3-color package (version 1.4.1) in the yarn-lock.json file, doesn't that reference need to be addressed as well?

Reply:
the yarn-lock.json is automatically managed by the yarn command, So I do not think is needed to be modified by hand.  I also think this file should be put in the .gitignore anyway.